pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -224,6 +230,8 @@ protected void updateTopicPolicyByNamespacePolicy(Policies namespacePolicies) {
                 type -> this.topicPolicies.getBackLogQuotaMap().get(type)
                         .updateNamespaceValue(MapUtils.getObject(namespacePolicies.backlog_quota_map, type)));
         updateSchemaCompatibilityStrategyNamespaceValue(namespacePolicies);
+        topicPolicies.getDispatchRate().updateNamespaceValue(
+            namespacePolicies.topicDispatchRate.get(brokerService.pulsar().getConfiguration().getClusterName()));
```

Comment:
Need to keep compatible with deprecated `clusterDispatchRate`.
It's used when `topicDispatchRate` is not set.

Reply:
> I think we need a unit test for this.

`org.apache.pulsar.broker.admin.TopicPoliciesTest#testPolicyOverwrittenByNamespaceLevel ` has covered  this @Jason918